src/main/resources/config.yml

```diff
@@ -53,6 +53,19 @@ afk:
     enabled: "&7AFK detection is now enabled."
     disabled: "&7AFK detection is now disabled."
 
+alias:
+  greeting:
+    no-pronouns:  "§7You have not set any pronouns. See §e/help pronouns"
```

Comment:
We now have a mix of `§` and `&` color codes. I'd prefer to stick with `&` since it's easier to type on most keyboards.

Reply:
I personally don't like & because it can be ambiguous, for example in the phrase "get me a bow&arrow", but i will change it back for consistency

Reply:
It seems to be the convention for Spigot plugins as far as I could see. I don't really mind either way, but I do like it being consistent.

Reply:
(by which I mean, if you want to convert everything to simdollars, that's fine by me too 😃)